### PR TITLE
initial upload of test functions

### PR DESCRIPTION
**Pull request by g-nym**
Adding Gwen's test functions

### CODE REVIEW

File: Gwen/tests/test_functions.py [hunk 1]
```diff
@@ -1,5 +1,201 @@
...
+# +
+lat0 =  30.23  ; lat1 = 42.135
+lon0 =  -114.139 ; lon1 = -177.269
+
+algae_extent = [lon0, lon1, lat0, lat1]
+
+# +
+lat0 =  49.36  ; lat1 = 24.55
+lon0 =  -67.00 ; lon1 = -124.77
+
```
Comment (Samlewis99) [inline Gwen/tests/test_functions.py:188]: Just checking if this code needs to be in the test functions page?

**Review by Samlewis99 — COMMENTED** [review]
Looks great Gwen, I just want to check one thing before you commit. _See the comment_

**Review by XuluLin — COMMENTED** [review]
I did not find any issues with this. I like how you tested the dimension of the numpy array as well.

File: Gwen/tests/test_functions.py (hunk 1 above)
Comment (Samlewis99) [reply Gwen/tests/test_functions.py:188]: Sorry just saw why these are in there! my bad disregard last comment

**Review by Samlewis99 — APPROVED** [review]
All looks good!

**Pull request merged**